knative-operator/deploy/resources/kn-downloads.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
```

Comment:
need suggestion if it should use this namespace

Reply:
I'm wondering if we can make this part of the CSV instead 🤔 . @jcrossley3 wdyt?

Reply:
@jcrossley3 ping

Reply:
I don't understand the question -- you want to make a namespace part of the CSV?

Reply:
There are two questions here:

1 - Which namespace to use to create this deployment, route, service into?
2 - Where to put all these resources YAMLs?

Reply:
We should definitely not create a new namespace. In the absence of any better ideas, it seems reasonable to me to just put this Deployment into the CSV and then OLM will automatically install it into whichever namespace the operator is installed into.

Reply:
The latest on the PR applies Deployment, Route and Service in KnativeServing instance's namespace.